index.html

```diff
@@ -0,0 +1,524 @@
+        <li class="navbar-list_link"><a href="#" class="n-d">Jobs</a></li>
+        <li class="navbar-list_link more active"><a href="#" class="n-d">More</a><img
+            src="https://www.smashingmagazine.com/images/icons/menu-cat.svg" class="cat" alt="cat"></li>
+      </ul>
+      <input type="search" class="searchbar" placeholder="Search articles..." />
+    </nav>
+  </header>
+
+  <div class="latest-articles d-flex c-fff ff-Jadi">
+    <div class="latest-articles_title ">Don't miss this latest</div>
+    <div class="latest-articles_subtitle">Articles</div>
+    <div class="article a-left c-fff">
+      <div class="art">
+        <div class="imgheading d-flex mid-fs"><img
+            src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/82c26edc-5500-42a6-87a9-62e84b998433/rachel-andrew-tw-profile.jpg"
+            class="profileimg" alt="person profile pic" />
+          <p>Rachael Andrew <span class="light">wrote</span></p>
+        </div>
+        <p class="text c-fff">Smashing Magazine is Fourteen! <span class="com">3 comments</span></p>
+        <p class="hashtags">
+          <span class="ht"># Events<sup class="sup">91</sup></span>
+          <span class="ht"># Smashing<sup class="sup">11</sup></span>
+          <span class="ht"># Smashing Conf<sup class="sup">33</sup></span>
+        </p>
+      </div>
+    </div>
+    <div class="article">
+      <div class="art">
+        <div class="imgheading d-flex mid-fs"><img
+            src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/82c26edc-5500-42a6-87a9-62e84b998433/rachel-andrew-tw-profile.jpg"
+            class="profileimg" alt="person profile pic" /> Katy Bowman<span class="light">wrote</span></div>
+        <p class="text">An Introduction to running Lighthouse Programmatically<span class="com">1 comment</span></p>
+        <p class="hashtags">
+          <span class="ht"># Performance<sup class="sup">131</sup></span>
+          <span class="ht"># Techniques<sup class="sup">241</sup></span>
+          <span class="ht"># Tools<sup class="sup">251</sup></span>
+        </p>
+      </div>
+    </div>
+    <div class="article a-left_bottom">
+      <div class="art">
+        <div class="imgheading d-flex mid-fs"><img
+            src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/82c26edc-5500-42a6-87a9-62e84b998433/rachel-andrew-tw-profile.jpg"
+            class="profileimg" alt="person profile pic" /> Jonathan Kelley<span class="light">wrote</span></div>
+        <p class="text">Desiging with reduced motion for motion sensitives<span class="com">Leave a comment</span></p>
+        <p class="hashtags">
+          <span class="ht"># API<sup class="sup"></sup></span>
+          <span class="ht"># CMS<sup class="sup"></sup></span>
+          <span class="ht"><sup class="sup"></sup></span>
+        </p>
+      </div>
+    </div>
+    <div class="article a-left">
```

Comment:
- [ ] Use ```article``` tag wherever is needed there are various lines of code that's need it so look around. Having a class article is perfect. The recommendation here is that if there is an HTML tag for certain types of content use it over a generic ```div``` tag or other generic HTML tags.